tools/tpm2_listpcrs.c

```diff
@@ -173,38 +173,36 @@ static bool read_pcr_values(listpcr_context *context) {
     return true;
 }
 
-static void init_pcr_selection_from_algorithm(TPMI_ALG_HASH alg_id,
-        TPML_PCR_SELECTION *pcr_selections) {
-
-    pcr_selections->count = 1;
-    pcr_selections->pcrSelections[0].hash = alg_id;
-    set_pcr_select_size(&pcr_selections->pcrSelections[0], 3);
-    clear_pcr_select_bits(&pcr_selections->pcrSelections[0]);
-
-    UINT32 pcr_id;
-    for (pcr_id = 0; pcr_id < 24; pcr_id++) {
-        set_pcr_select_bit(&pcr_selections->pcrSelections[0], pcr_id);
+static bool init_pcr_selection(TPMI_ALG_HASH alg_id, listpcr_context *context)
+{
```

Comment:
Currently this code base is a mixed mess of style, but adhere to the coding style under: tpm2.0-tools/misc/coding_standard_c.txt.

Functions have opening brace on the same line, variables use underscores, no camel case.